origami.json

```diff
@@ -34,6 +34,11 @@
 				"description": "Display on click demo"
 			},
 			{
+				"name": "fixed",
+				"template": "demos/src/fixed.mustache",
+				"description": "Fixed positioned tooltip demo"
```

Comment:
I think this demo should be hidden – we tend to hide demos which, in the registry, look no different from each other. This one's is mostly useful for testing fixed position stuff locally rather than as a demonstration of how the tooltip works